test/studies/prk/stencil/stencil-serial-fast.perfkeys

```diff
@@ -0,0 +1 @@
+Rate (MFlops/s): 
```

Comment:
I would also be tempted to gather the time as well. Note that if you add it here, it'll show up in the .dat file in case we want to use it in the future, but it won't be on the graph (you'd have to add it to the .graph file too)